schema_based_on_diagram.sql

```diff
@@ -0,0 +1,51 @@
+CREATE TABLE patient(
+ id SERIAL PRIMARY KEY NOT NULL,
+ name VARCHAR(60),
+ date_of_birth DATE
+);
+
+CREATE TABLE medical_histories(
+ id SERIAL PRIMARY KEY NOT NULL,
+ admited_at TIMESTAMP,
+ patient_id INT REFERENCES patient(id),
```

Comment:
[OPTIONAL]
Same here:
```suggestion
 patient_id INT REFERENCES patients(id),
```